### ISSUE TITLE
Log File Monitoring

### ISSUE THREAD

**Issue by Santu439** [newrelic/newrelic-unix-monitor#41]
Is it feasible to do log file monitoring with pattern search feasible using this plugin ? Please advise

**Comment by Steve24by7** [comment]
Yes you can write a specification that runs the grep and parses the output to upload to New Relic.

**Comment by Santu439** [comment]
> Yes you can write a specification that runs the grep and parses the output to upload to New Relic.

Thank you. Sorry for delayed reply. If I use grep then even old logs of the file also may come as we may not have track of watermark on the file. We used splunk for now to monitor logs.

**Comment by Steve24by7** [comment]
Not enough information in this thread for me to know what you need.
